mono/metadata/Makefile.am

```diff
@@ -185,6 +185,16 @@ libmono_system_native_la_SOURCES = \
 
 libmono_system_native_la_CFLAGS = -I$(abs_top_srcdir)/external/corefx/src/Native/Unix/Common -I$(abs_top_srcdir)/external/corefx/src/Native/Unix/System.Native -Wno-typedef-redefinition
 
+if HOST_DARWIN
+lib_LTLIBRARIES += libmono-apple-crypto.la
```

Comment:
I don't feel strongly about it but should we name this `libmono-system-security-cryptography-native-apple.la`?

It'd better match the corefx naming and also what we did with `System.Native -> libmono-system-native.la`